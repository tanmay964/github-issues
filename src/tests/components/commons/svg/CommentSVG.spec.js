import React from 'react';
import renderer from 'react-test-renderer';
import CommentSVG from '../../../../components/commons/svg/CommentSVG';

it('renders correctly', () => {
  const tree = renderer
    .create(<CommentSVG />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
