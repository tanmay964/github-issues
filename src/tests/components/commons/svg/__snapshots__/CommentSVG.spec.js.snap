// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  aria-hidden="true"
  className="sc-bdVaJa ijurAl"
  height="16"
  width="16"
>
  <path
    d="M14 1H2c-.55 0-1 .45-1 1v8c0 .55.45 1 1 1h2v3.5L7.5 11H14c.55 0 1-.45 1-1V2c0-.55-.45-1-1-1zm0 9H7l-2 2v-2H2V2h12v8z"
    fillRule="evenodd"
  />
</svg>
`;
